docs/api-reference/mapbox/overview.md

```diff
@@ -14,7 +14,7 @@ Use deck.gl layers as custom Mapbox layers, enabling seamless interleaving of Ma
 
 ### Limitations
 
-* deck.gl's multi-view system cannot be used.
+* deck.gl's multi-view system only supports one MapView (e.g. minimaps are not supported).
```

Comment:
Technically speaking you can have multiple `MapView`s, the limitation is that you will always have a MapView that fills the entire canvas (what Mapbox draws into).

Reply:
Ah, is that why minimap views can't use mapbox? I'd like to explain the limitation better in the overview in case its useful to someone hacking around it.